### ISSUE TITLE
Some class names out of sync between v4 docs and actual CSS

### ISSUE THREAD

**Issue by iqnivek** [twbs/bootstrap#20111]
The v4 docs say to use `btn-*-outline` (e.g. `btn-primary-outline`) for outline buttons: http://v4-alpha.getbootstrap.com/components/buttons/#outline-buttons. However, the v4.0.0-alpha.2 CSS uses the format `btn-outline-*` (e.g. `btn-outline-primary`) instead: https://cdn.rawgit.com/twbs/bootstrap/v4-dev/dist/css/bootstrap.css)

Similarly, the class names are out of sync for the grid offset classes. The docs say to use `col-*-offset-*` (e.g. `col-md-offset-4`): http://v4-alpha.getbootstrap.com/layout/grid/#example-offsetting-columns, while the CSS has classes of the form `offset-*-*`.

I'm happy to make a PR, but I wasn't sure if the docs are correct, or the CSS.

**Comment by cvrebert** [comment]
Duplicate of #19899. Please search more next time. Thanks!

**Comment by cvrebert** [comment]
TL;DR: v4-dev != v4.0.0-alpha.2. The latter is a tagged release version, the former is an active development branch. The hosted docs reflect the latter. Changes have been made in the branch since the release was shipped. They're NOT interchangeable.

**Comment by iqnivek** [comment]
understood, thanks for explaining. I did do a search but could've looked more carefully. cheers.
